elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -297,13 +297,17 @@ class PfeJumpLinksPanel extends PFElement {
     this._mutationCallback = this._mutationCallback.bind(this);
     this._observer = new MutationObserver(this._mutationCallback);
     this.currentActive = null;
-    this.sectionMargin = this.getAttribute("offset") || 200;
     this.currentActive = 0;
     this.current = -1;
+    this._getNav = this._getNav.bind(this);
     this.nav = this._getNav();
+    this.offset = this.nav.getAttribute("offset") || 200;
```

Comment:
You can query for custom properties using the same var function, ` this.var(offset)`